src/utils/signaling.js

```diff
@@ -281,16 +280,9 @@ Signaling.Base.prototype.joinCall = function(token, flags, silent, recordingCons
 			}.bind(this))
 			.catch(function() {
 				reject(new Error())
-				if (!IS_DESKTOP) {
-					// Server maintenance, lobby kicked in, or room not found.
-					// We first redirect to the conversation again and that
-					// will then show the proper error message to the user.
-					window.location = generateUrl('call/' + token)
-				} else {
-					// TODO: Is it true, reload is equal to generateUrl('call/' + token) here?
-					// Or can we always just reload the page?
-					window.location.reload()
-				}
+				store.commit('connectionFailed', {
+					token,
+				})
```

Comment:
If we can get the meaningful error message from `axios.post` request and pass it to here, then it would be perfect.

Even if we get `error => 'consent'`, should be possible to interprete response and show relevant text as iOS does it